jdaviz/configs/default/plugins/export/tests/test_export.py

```diff
@@ -183,3 +183,36 @@ def test_basic_export_subsets_cubeviz(self, tmp_path, cubeviz_helper, spectral_c
         with pytest.raises(NotImplementedError,
                            match='Subset can not be exported - Export for composite subsets not supported.'):  # noqa
             export_plugin.export()
+
+
+def test_export_data(cubeviz_helper, spectrum1d_cube):
+    cubeviz_helper.load_data(spectrum1d_cube, data_label="test")
+    export_plugin = cubeviz_helper.plugins["Export"]._obj
+    export_plugin.dataset_selected = 'test[FLUX]'
```

Comment:
can you check that the data_invalid_msg is set correctly in both of these tests?